applications/concept-catalogue/src/main/kotlin/no/begrepskatalog/conceptregistration/storage/SqlStore.kt

```diff
@@ -516,7 +516,12 @@ class SqlStore(val connectionManager: ConnectionManager) {
         throw RuntimeException("While mapping statuses, encountered status $statusFromDb , that is not one of (1(UTKAST),2(GODKJENT),3(PUBLISERT))")
     }
 
-    fun mapForholdTilKilde(forholdFromDb: String? ) : Kildebeskrivelse.ForholdTilKildeEnum {
+    fun mapForholdTilKilde(forholdFromDb: String? ) : Kildebeskrivelse.ForholdTilKildeEnum? {
```

Comment:
Du kan endre denne, og de som er like, til when, i stedet for rekken med if

````
return when (forholdFromDb) {
        null -> null
        1.toString() -> Kildebeskrivelse.ForholdTilKildeEnum.EGENDEFINERT
        2.toString() -> Kildebeskrivelse.ForholdTilKildeEnum.BASERTPAAKILDE
        3.toString() -> Kildebeskrivelse.ForholdTilKildeEnum.SITATFRAKILDE
        else -> throw RuntimeException
    }

Reply:
Ah, så mye bedre lesbarhet!